### DISCUSSION TITLE
How to enable vue defineModel experimental feature?

### DISCUSSION THREAD

**Discussion by farr0202** [quasarframework/quasar#16245] [General - Components / Directives / etc] [answered]
Hello,
I want to use the new Vue3.3 [defineModel](https://github.com/vuejs/rfcs/discussions/503) feature in my quasar app but have no idea how to activate it (as it requires an explicit opt-in). Unfortunately the answers given in #12450 do not work. (My project uses Quasar CLI with Webpack.)
Anyone can give me a hint to get it working?

Bye
   Holger

**Comment by metalsadman** [comment]
https://github.com/vuejs/rfcs/discussions/503
```
// vite.config.js
export default {
  plugins: [
    vue({
      script: {
        defineModel: true
      }
    })
  ]
}
```

https://discord.com/channels/415874313728688138/902518756436226079/1138404587406102578

> quasar.config file > build > viteVuePluginOptions >
`script: {
  defineModel: true
}`

**Reply by farr0202** [reply]
Thank you for the answer, your solution works when I use Quasar CLI with Vite.

But is there a solution for using Quasar CLI with Webpack. I tried to set `quasar.conf.js > build > vueLoaderOptions: { defineModel: true }` as mentioned by [Razvan Stoenescu](https://github.com/rstoenescu) in #12450 but that doesn't work.

**Reply by yusufkandemir** [reply, marked as answer]
@farr0202 the latest app-webpack version uses `vue-loader` 17.0.1: https://github.com/quasarframework/quasar/blob/aed7a6ad15d4c822beee594d61c97d994ffab500/app-webpack/package.json#L102
while the requirement for `defineModel` is ^17.1.1: https://github.com/vuejs/rfcs/discussions/503#:~:text=)%0A%20%20%5D%0A%7D-,vue%2Dcli,-Requires%20vue%2Dloader

It's updated for the upcoming major CLI version, but the current major version doesn't have it.
/cc @rstoenescu

**Reply by farr0202** [reply]
@yusufkandemir Good point :-)

**Reply by rstoenescu** [reply]
Pushed a fix. Will make a new q/app-webpack release later today.
